gitlab/v4/objects/epics.py

```diff
@@ -29,6 +30,63 @@ class GroupEpic(ObjectDeleteMixin, SaveMixin, RESTObject):
     resourcelabelevents: GroupEpicResourceLabelEventManager
     notes: GroupEpicNoteManager
 
+    def _epic_path(self) -> str:
+        """Return the API path for this epic using its real group."""
+        if not hasattr(self, "group_id") or self.group_id is None:
```

Comment:
Can a `group_id` actually be none for an epic, since it's always a group resource? Or can we go with a simple try/except here?

One other thought - will this work if we get an epic with `lazy=True` from an arbitrary group?